test/adapter/polymorphic_test.rb

```diff
@@ -165,6 +165,53 @@ def test_json_api_serialization
 
           assert_equal(expected, serialization(@picture, :json_api))
         end
+
+        def test_json_api_serialization_with_polymorphic_belongs_to
```

Comment:
I added this test and ran it against the v0.10.5 tag and it passed. I then worked on 0-10-stable until it passed again